public/js/services/interceptor.js

```diff
@@ -0,0 +1,13 @@
+angular.module('mean.system')
+  .factory('AuthInterceptor', ['LocalStorage', function(LocalStorage) {
+    var AuthInterceptor = {
```

Comment:
careful when you are accepting hounds let and var suggestions, let scopes your variable definition and might cause issues with the global this.